src/XrdOuc/XrdOucUtils.hh

```diff
@@ -130,7 +131,36 @@ static bool PidFile(XrdSysError &eDest, const char *path);
 
 static int getModificationTime(const char * path, time_t & modificationTime);
 
-       XrdOucUtils() {}
-      ~XrdOucUtils() {}
+    //------------------------------------------------------------------------
```

Comment:
It appears (I may be wrong) that this was inlined in the "hh" file. Could you move the actual implementation to the "cc" file to keep the "hh" file reasonably clean.